### ISSUE TITLE
Up-down wall sconces

### ISSUE THREAD

**Issue by LarsStegman** [Templarian/MaterialDesign#7501]
### Before submitting a new icon request, please confirm the following:

- [X] I have [searched all issues](https://github.com/Templarian/MaterialDesign/issues?q=is%3Aissue) to make sure there isn't a request for this icon.
- [X] I have [searched the current library](https://pictogrammers.com/library/mdi) to make sure the icon doesn't already exist.
- [X] I understand that [brand icons are no longer being added](https://pictogrammers.com/docs/library/mdi/guides/brand-icons/) and I can use [SimpleIcons](https://simpleicons.org/) for my brand icon needs.
- [X] I only intend to request a single icon (or a few near-identical ones) in this issue.

### What would you use this icon for?

Home assistant

### Give us an example of what this icon might look like.

A wall sconce variant that points light up and down. 

Like these sconces from IKEA https://www.ikea.com/nl/nl/p/nymane-wandlamp-up-down-vaste-inst-antraciet-20415211/

I imagine it would look something like this
<img width="693" alt="image" src="https://github.com/user-attachments/assets/47969720-a5b5-4c7d-a44d-71c3bfd23667">

**Comment by steodor** [comment]
There wouldn't be any `outline` since the on/off state cannot be represented via the "beams" as you drew them. Maybe I'm wrong (please correct me) but I haven't seen any icon with hollowed-out short beams. So then on/off would be depicted as the presence or absence of the beams plus the hollow part of the sconce. See if these fit your needs:

![image](https://github.com/user-attachments/assets/4ed6f04e-c536-4111-b7c7-ca8543d02a30)
[wall-sconce-double-on.zip](https://github.com/user-attachments/files/16649435/wall-sconce-double-on.zip)

![image](https://github.com/user-attachments/assets/a87b3d67-998f-4c99-a206-97676364beb0)
[wall-sconce-double-on-up.zip](https://github.com/user-attachments/files/16649437/wall-sconce-double-on-up.zip)

![image](https://github.com/user-attachments/assets/b4ef9ed7-a861-44f1-9dfc-310ecd78118d)
[wall-sconce-double-on-down.zip](https://github.com/user-attachments/files/16649438/wall-sconce-double-on-down.zip)

![image](https://github.com/user-attachments/assets/9d57b88b-78e8-437d-8911-afa547f2857c)
[wall-sconce-double-off.zip](https://github.com/user-attachments/files/16649439/wall-sconce-double-off.zip)

**Comment by LarsStegman** [comment]
That looks good! I based my sketch on the wall-sconce-flat-variant icon, so they should probably be called after that one. 

https://pictogrammers.com/library/mdi/icon/wall-sconce-flat-variant/

**Comment by steodor** [comment]
Yeah, I thought it would be too long, but you can rename them at will of course.

**Comment by edcoppen** [comment]
I came here to request exactly this. In my opinion we need: 

mdi:wall-sconce-flat-up showing light pointing up
mdi:wall-sconce-flat-down showing light pointing down 
mdi:wall-sconce-flat-both showing light pointing up and down
